.github/workflows/source_actions_exist.yaml

```diff
@@ -0,0 +1,43 @@
+name: "Workflow_source_exists"
```

Comment:
we don't need this. Can just be part of normal unit tests

Reply:
The github workflow? If I put it in the unit tests then it will work with `testthat` but with `devtoold::check()` and on the current github checks the test fails as it can't find the .github directory, as this isn't built in the package just the repo.